ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh

```diff
@@ -1,39 +1,37 @@
 #!/bin/bash
 
 if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
-    source "${SHARED_DIR}/proxy-conf.sh"
+  source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
 pwd && ls -ltr
 cd frontend || exit 0
 pwd && ls -ltr
 
 ## skip all tests when console is not installed
-ret=0
-oc get co console --kubeconfig=${KUBECONFIG} || ret=$?
-if [ $ret -ne 0 ]; then
+if (oc get clusteroperator console --kubeconfig=${KUBECONFIG}) ; then
```

Comment:
I tried this if condition locally against normal cluster, the result is:
```
NAME      VERSION                              AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
console   4.15.0-0.nightly-2023-10-24-230302   True        False         False      129m    
console is not installed, skipping all console tests.
```
should it use the opposite of the result here?  "!"

Reply:
:+1:    Fixed.